abixen-platform-business-intelligence-service/src/main/java/com/abixen/platform/service/businessintelligence/multivisualisation/model/impl/database/DatabaseConnection.java

```diff
@@ -157,5 +157,93 @@ public void setPassword(String password) {
         this.password = password;
     }
 
+    public static DatabaseConnectionBuilder databaseConnectionBuilder() {
+        return new DatabaseConnectionBuilder();
+    }
+
+    public static final class DatabaseConnectionBuilder {
```

Comment:
Make the builder similarly like in ddd-core branch.
here can be folloing methods:
credentials
database
details